src/components/inputs/input-text/types.ts

```diff
@@ -7,6 +7,12 @@ import { IconsName } from '../../../primitives/icon/icon.types';
 
 export type Root = NativeTextInput | undefined | null;
 
+export enum InputType {
```

Comment:
I would recommend to not do an enum but instead 

```tsx
'TEXT' | 'SEARCH' | 'PHONE'
```

This way, we won't need to import `InputType`, but rather use directly `'PHONE'` for instance, as we do with Appearance and Colors also.